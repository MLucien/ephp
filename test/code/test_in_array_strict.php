<?php
$a = array('1.10', 12.4, 1.13);

if (in_array('12.4', $a, true))
    print "Found '12.4' strict\n";

if (in_array('12.4', $a, false))
    print "Found 12.4 no strict\n";

if (in_array(1.13, $a, true))
    echo "Found 1.13 strict\n";

var_dump(in_array(1, $a, 25));
var_dump(in_array(1, $a, array()));
