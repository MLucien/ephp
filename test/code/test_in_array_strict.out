Found 12.4 no strict
Found 1.13 strict
bool(false)

Warning: in_array() expects parameter 3 to be boolean, array given in {{CWD}}/test/code/test_in_array_strict.php on line 14
NULL
